src/.clang-format

```diff
@@ -18,7 +18,6 @@ AlignOperands:   false
 AlignTrailingComments: true
 AllowShortBlocksOnASingleLine: true
 AllowShortIfStatementsOnASingleLine: true
-AllowShortBlocksOnASingleLine: true
```

Comment:
We shouldn't change this happhazzerdly, it's in sync across all our repos so should decide collectively to change. Is this what sisl, homestore etc have?

Reply:
@szmyd I removed the verbose of Line 19.